docker/Dockerfile

```diff
@@ -4,10 +4,10 @@ ARG REGISTRY=${REGISTRY:-registry.mapotempo.com/}
 ARG VROOM_VERSION=${VROOM_VERSION}
 
 # Install Vroom
-FROM ${REGISTRY}mapotempo/vroom:${VROOM_VERSION} as vroom
+FROM ${REGISTRY}mapotempo/vroom:${VROOM_VERSION:-v1.5.0} as vroom
 
 # Rake
-FROM ${REGISTRY}mapotempo-${BRANCH}/optimizer-ortools:${OPTIMIZER_ORTOOLS_VERSION}
+FROM ${REGISTRY}mapotempo-${BRANCH}/optimizer-ortools:${OPTIMIZER_ORTOOLS_VERSION:-v1.4.0}
```

Comment:
There is a file called `version.sh` which also has the default for `OPTIMIZER_ORTOOLS_VERSION` environment variable. 

So the default of optimizer-ortools will be in two places at the moment. I wonder if there is an easy way to simplify this while keeping the functionality coming from there (i.e., devs can change the OPTIMIZER_ORTOOLS_VERSION if they want)..

@alain-andre if we modify the content of `version.sh` as follows, Docker image generation and Travis should continue to work correctly, right ?
```bash
#!/usr/bin/env bash

if ! { [[ $TRAVIS_BRANCH == "master" ]] || [[ $TRAVIS_TAG != "" ]]; }; then
  export OPTIMIZER_ORTOOLS_VERSION=${OPTIMIZER_ORTOOLS_VERSION:-Mapotempo};
fi
```

Reply:
Yes @senhalil you are right. But I would revert the conditions to set default `OPTIMIZER_ORTOOLS_VERSION` to `Mapotempo` only when *$TRAVIS_BRANCH* is not *master* or *$TRAVIS_TAG* is empty. It was written by @braktar he could have an idea.

Reply:
I just forgot to inverse the middle OR :facepalm: ! 

The correct condition is as follows
```bash
if ! { [[ $TRAVIS_BRANCH == "master" ]] && [[ $TRAVIS_TAG != "" ]]; }; then
  export OPTIMIZER_ORTOOLS_VERSION=${OPTIMIZER_ORTOOLS_VERSION:-Mapotempo};
fi
```

which keeps the same functionality of the current code (below) while moving the default (1.4.0) to docker file.

```bash
if [[ $TRAVIS_BRANCH == "master" ]] || [[ $TRAVIS_TAG != "" ]]; then
  export OPTIMIZER_ORTOOLS_VERSION="v1.4.0";
else
  export OPTIMIZER_ORTOOLS_VERSION=${OPTIMIZER_ORTOOLS_VERSION:-Mapotempo};
fi
```

Reply:
wow, I did not noticed `! { ... }`. I change my PR.